From 5720ff12d4a895824070d2a53e8220e631ac351e Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Tue, 8 Sep 2015 15:25:42 +0800
Subject: [PATCH 1079/1221] MLK-11520 ARM: imx: enable axi clk in clk_init_on
 on i.mx6qdl

AXI clk should be always enabled, as many peripheral
devices need this clk to be on, especially for busfreq,
AXI podf change needs AXI clk and its parent on.

Signed-off-by: Anson Huang <b20788@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/mach-imx/clk-imx6q.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-imx/clk-imx6q.c b/arch/arm/mach-imx/clk-imx6q.c
index 2de743c..3cdbaf1 100644
--- a/arch/arm/mach-imx/clk-imx6q.c
+++ b/arch/arm/mach-imx/clk-imx6q.c
@@ -106,6 +106,7 @@ static unsigned int const clks_init_on[] __initconst = {
 	IMX6QDL_CLK_ROM,
 	IMX6QDL_CLK_ARM,
 	IMX6QDL_CLK_OCRAM,
+	IMX6QDL_CLK_AXI,
 };
 
 static struct clk_div_table clk_enet_ref_table[] = {
-- 
1.7.5.4

